.rubocop_todo.yml

```diff
@@ -0,0 +1,7 @@
+# This configuration was generated by
```

Comment:
Is this file used?

Reply:
It is required by the rubocop.yml. It is empty right now because I cleaned up the issues that Rubocop generated when I ran it. If we update Rubocop and/or add more rules, it is a good way to fix incrementally.

Reply:
I'm curious why you left it behind since its serving no purpose at this point?

I would like to think this file would be added as needed, along with a good reason that the violations were not resolved at the time they were added.

For example, if a new Cop was added... I assume you may do this as the teams "standards" change. New projects would get this new rubocop configuration, but existing projects would only address outstanding violations as those parts of the code were touched.

Long story short, I don't think this file should be included. :smile:

Reply:
I can take it out. I am used to run in my local machine ```rubo --auto-gen-config``` and then go ahead and knock off the necessary file.